content/en/real_user_monitoring/heatmaps/_index.md

```diff
@@ -33,7 +29,7 @@ datadogRum.init({
 
 ### Getting started
 
-On the [heatmap page][3], select your application and view. A heatmap displays the aggregrate of where all users (sessions) clicked on a single view, with the view being a single page within an application. Each session is one user's activity and includes everything they did, including all pages they viewed and all actions they clicked.
+On the view list page, select your application and view. This will take you to the[heatmap page][3]for a particular view. You can switch the view being shown with the view and application selector at the top.A heatmap displays the aggregrate of where all users (sessions) clicked on a single view, with the view being a single page within an application. Each session is one user's activity and includes everything they did, including all pages they viewed and all actions they clicked.
```

Comment:
```suggestion
On the view list page, select your application and view. This will take you to the [heatmap page][3] for a particular view. You can switch the view being shown with the **View Name** and **Application** selectors at the top. A heatmap displays the aggregate of where all users (sessions) clicked on a single view, with the view being a single page within an application. Each session is one user's activity and includes everything they did, including all pages they viewed and all actions they clicked.
```